truffleruby/src/main/ruby/core/rbconfig.rb

```diff
@@ -148,6 +144,11 @@ def RbConfig.expand(val, config = CONFIG)
     val.replace(newval) unless newval == val
     val
   end
+
+  CONFIG.each_value do |val|
+    RbConfig::expand(val)
+  end
```

Comment:
Why can't `RbConfig` been lazily loaded? It's behind `require 'rbconfig'` normally anyway isn't it?